loader/schemas.py

```diff
@@ -0,0 +1,232 @@
+                                       "will be assumed.\n"
+                                       "\n"
+                                       "possible values:\n"
+                                       "md5                # most blob stores provide a checksum using this\n"
+                                       "multipart-md5      # multipart uploads provide a specialized tag in S3\n"
+                                       "sha256\n"
+                                       "sha512"
+                    }
+                }
+            },
+
+            'description': 'REQUIRED\n'
+                           'The checksum of the Data Object. At least one checksum must be provided.'
+        },
+        'urls': {
+            'type': 'array',
+            'items': {
+                "type": "object",
+                "properties": {
+                    "url": {
+                        "type": "string",
+                        "description": "REQUIRED\n"
+                                       "A URL that can be used to access the file."
+                    },
+                    "system_metadata": {
+                        "type": "object",
+                        "additionalProperties": True,
+                        "description": "OPTIONAL\n"
+                                       "These values are reported by the underlying object store.\n"
+                                       "A set of key-value pairs that represent system metadata about the object."
+                    },
+                    "user_metadata": {
+                        "type": "object",
+                        "additionalProperties": True,
+                        "description": "OPTIONAL\n"
+                                       "A set of key-value pairs that represent metadata provided by the uploader."
+                    }
+                }
+            },
+            'description': 'OPTIONAL\n'
+                           'The list of URLs that can be used to access the Data Object.'
+        },
+        'description': {
+            'type': 'string',
+            'description': 'OPTIONAL\n'
+                           'A human readable description of the contents of the Data Object.'
+        },
+        'aliases': {
+            'type': 'array',
+            'items': {
+                'type': 'string'
+            },
+            'description': "OPTIONAL\n"
+                           "A list of strings that can be used to find this Data Object.\n"
+                           "These aliases can be used to represent the Data Object's location in\n"
+                           "a directory (e.g. \'bucket/folder/file.name\') to make Data Objects\n"
+                           "more discoverable."
+        }
+    }
+}
+
+data_bundle = {
+    "type": "object",
+    # TODO: required, but we don't have checksums for bundles
+    'required': ['data_object_ids', 'created', 'updated', 'version'],  # , 'checksums'],
```

Comment:
Ditto comment above regarding consistency between the values of `required` here and the use of "OPTIONAL" and "REQUIRED" in the descriptions.